src/terra-dev-site/guide/GeneralGuidelines.l.guide.mdx

```diff
@@ -0,0 +1,8 @@
+# General Guidelines
+
+Keep in mind these things when developing terra components
+
+* Terra components are black boxes. You should not rely on internal implementation specifics. This includes DOM structure, classes, and undocumented side effects.
+* Try and keep your DOMs small. If you have a list that could have thousands of items in it, you might need to rethink your design
+* Take some time to learn the basics of JavaScript and React
```

Comment:
Links to resources might be helpful

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript